app/src/main/java/fr/free/nrw/commons/upload/mediaDetails/UploadMediaDetailFragment.java

```diff
@@ -246,6 +246,13 @@ public void showSimilarImageFragment(String originalFilePath, String possibleFil
             public void onPositiveResponse() {
                 Timber.d("positive response from similar image fragment");
                 presenter.useSimilarPictureCoordinates(similarImageCoordinates, callback.getIndexInViewFlipper(UploadMediaDetailFragment.this));
+
+                // set the description text when user selects to use coordinate from the other image
```

Comment:
@nicolas-raoul Is it ok to write comments here like this? 

Looks good to me otherwise, the PR solves the issue as expected.

Reply:
@Ayan-10 Yes writing comments in code is always a good idea. The issue links gives some extra context to readers who would want extra context. Thanks!